.gitignore

```diff
@@ -20,4 +20,8 @@
 # macOS
 *.DS_store
 
+# Editors
+.vscode
```

Comment:
Generally speaking, I recommend that folks add their editor of choice's files to their personal global git ignore instead of expecting every project they work with to do so individually.

That said, we do already have two other editor files here, at L8 and L9. I'd be ok with adding this heading and additional file up there.